app/models/category.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Category < ApplicationRecord
+  # Include Discard gem
```

Comment:
In the case of Ruby and Rails it's not recommended to add comments to the code unless they are necessary.

Reply:
im working on it thanks for rview